README.md

```diff
@@ -182,17 +182,25 @@ Optional steps:
 [bash completions]: extra/completions/liquidctl.bash
 
 
-## Installing on FreeBSD
+## Installing on FreeBSD or DragonFlyBSD
 
-liquidctl is maintained in the Ports Collection (thanks to ehaupt@FreeBSD.org), and it is available as a pre-built binary package.
+### FreeBSD
+
+liquidctl is maintained in the FreeBSD Ports Collection (thanks to ehaupt@FreeBSD.org), and it is available as a pre-built binary package.
 
 - port: `sysutils/py-liquidctl`
 - binary: `pkg install py37-liquidctl`
 - dependencies: `devel/py-docopt`, `comms/py-hidapi`, `devel/py-pyusb`
 
 By default, root privileges (`doas` or `sudo`) are required to run liquidctl.
 
-Note: as of March 20, 2021, HIDAPI (`comms/hidapi`), [ ... ]
+To gain full access as a normal user without `doas` or `sudo`, see devd(8) (FreeBSD-specific guide needed). Also, you might consider manually changing the permission of the file of the USB device for an individual session with `chown`, e.g. `sudo chown [user] /dev/ugen[#.#]`.
+
+Note: as of March 20, 2021, HIDAP (`comms/hidapi`), which is required for `comms/py-hidapi` (and, thus, liquidctl), must be built from the latest port rather than installed as a package, as the latest package is out of date. Make sure that you have HIDAPI version 0.10.1 or later installed prior to installing `comms/py-hidapi` and liquidctl. Then, liquidctl will work as expected.
+
+### DragonFlyBSD
+
+liquidctl is available in DragonFly Ports, having been transferred from the FreeBSD Ports Collection. Installation and use should be the same as in FreeBSD (verification needed).
```

Comment:
Do you know _for a fact_ that it was transferred from the FreeBSD collection?  Just because that can happen, doesn't necessarily mean that it happened in this case.

You went to the trouble of thanking ehaupt above... what if ehaupt really did create and test ports for both?

Then, this doesn't really help the user:

```diff
-Installation and use should be the same as in FreeBSD (verification needed).
```

I would simply remove this DragonFly subsubsection (and the heading for the FreeBSD only subsubsection) and just mention DragonFly in the "Installing ..." subsection name.

By the way, you also need to fix the table of contents.

Reply:
I am 99.9% certain, as it even says in the DragonFly BSD port file itself "$FreeBSD$", etc., etc. But I think that I found a softer way of simply mentioning the DragonFly availability.